doc/source/conf.py

```diff
@@ -157,6 +157,10 @@
     "_build",
     "source/workflows/api/doc/ray.workflow.*",
     "source/serve/api/doc/ray.serve.*",
+    # exclude ray.remote type checking overloads from docs.
+    # refer to https://github.com/ray-project/ray/pull/33243
+    # for more context.
+    "../python/ray/_private/worker.pyi",
```

Comment:
why `pyi` instead of normal `py` file?

Reply:
looks that's the convention of writing interfaces.. and it was worker.pyi and removed by https://github.com/ray-project/ray/pull/25439